scripts/code_coverage_report/get_diff_to_json.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# USAGE: git diff -U0 [base_commit] HEAD | get_diff_lines.sh [output_file_name]
```

Comment:
Please change usage description to fix the naming and remove the `[output_file_name]`